src/app/resources/view-resources/resources-view.component.html

```diff
@@ -9,6 +9,10 @@
     <button mat-button (click)="toggleFullView()" i18n>
       {fullView, select, on {Show} off {Hide}} details
     </button>
+    <span class="toolbar-fill"></span>
+    <a href={{resourceSrc}} mat-icon-button download="{{resource.filename}}" [disabled]="!resource.isDownloadable">

```

Comment:
Setting the download attribute changes the file name to be saved locally, so by setting it to what the file is already named you aren't doing anything.